docs/dev/table/functions/systemFunctions.md

```diff
@@ -34,6 +34,8 @@ Scalar Functions
 
 The scalar functions take zero, one or more values as the input and return a single value as the result.
 
+Note: You can 
```

Comment:
Something's missing here ;)

Reply:
yes, the line should not be present anymore